src/strategies/DSL/conditions/nonceCondition.ts

```diff
@@ -1,11 +1,32 @@
 import { PrimitiveArgs } from '@brinkninja/types'
-import { NonceConditionArgs } from '@brink-sdk'
+import { NonceConditionArgs, nonceToBit, BitJSON } from '@brink-sdk'
 
-function nonceCondition (def: NonceConditionArgs): PrimitiveArgs[] {
+const STATE_USED = 'USED'
+const STATE_NOT_USED = 'NOT_USED'
 
-  // TODO: implement
+function nonceCondition ({state, nonce} : NonceConditionArgs): PrimitiveArgs[] {
+  const bit = nonceToBit({nonce})
 
-  return []
+  const bitJson: BitJSON = { index: bit.index.toString(), value: bit.value.toString() }
+
+  switch (state) {
+    case STATE_USED:
+      return [
+        {
+          functionName: 'requireBitUsed',
```

Comment:
I hadn't actually added primitive classes for the `requireBitUsed` and `requireBitNotUsed` functions yet, so I just did that and merged them to v5. You should rebase v5 branch so these are supported.

params expected are index and value. See how it's set up in tests here https://github.com/brinktrade/brink-sdk/blob/41c34a0b1bae8d5ad08314a7437de78fdda2af8b/test/local/Strategies.test.ts#L330